### PR TITLE
change strings on the dialog in the last level

### PR DESCRIPTION
**Pull request by laurelfan**
Here's the original mockup for this:

I kind of went with this but adapted it to use only existing strings so we wouldn't depend on another translation pass (that is also why some of them say Done and some say Finish...).

Playlab:
Before:

After:

Starwars:
Before:

After:

Maze:
Before:

After:

Minecraft:
Before:

After:

### CODE REVIEW

File: apps/src/feedback.js [hunk 1]
```diff
@@ -149,6 +149,9 @@ FeedbackUtils.prototype.displayFeedback = function(options, requiredBlocks,
 
   feedback.className += canContinue ? " win-feedback" : " failure-feedback";
 
+  var finalLevel = (options.response &&
+                    (options.response.message == "no more levels"));
```
Comment (Bjvanminnen) [inline apps/src/feedback.js]: nit: indent should only be two spaces, === instead of ==

Comment (Bjvanminnen) [inline apps/src/feedback.js]: will this be the response message for non-english users?

File: apps/src/feedback.js [hunk 2]
```diff
@@ -157,7 +160,8 @@ FeedbackUtils.prototype.displayFeedback = function(options, requiredBlocks,
       continueText: options.continueText,
       showPreviousButton: options.level.showPreviousLevelButton,
       isK1: options.level.isK1,
-      freePlay: options.level.freePlay
+      freePlay: options.level.freePlay,
+      finalLevel: finalLevel
```
Comment (Bjvanminnen) [inline apps/src/feedback.js]: could/should we just set continueText here instead of adding finalLevel?

**Comment by Bjvanminnen** [comment]
This seems okay for now. Should we maybe open a pivotal item somewhere to come up with a cleaner fix post-hoc?

**Comment by laurelfan** [comment]
This stuff is being tracked in a google doc, there are a bunch of other things that are not going to make hoc:

https://docs.google.com/document/d/1-N53Jqi1CsgAlSjlB-cm84xw84FNLaBlqfBiGXbw27k/edit#
https://docs.google.com/presentation/d/1PcdF_jm5gkru6Dslg3PbEkgPoJRnjxo5ktt8pAq551k/edit#slide=id.p3

**Comment by breville** [comment]
lgtm after chatting with laurel, who will modify the english string "final stage" to "final puzzle" which is more applicable to hour of code scripts.

**Comment by laurelfan** [comment]
verified with @tanyaparker that changing the .en string won't invalidate the existing translated strings

**Comment by philbogle** [comment]
Could you make sure that this is tested in the bug bash on Friday?

**Comment by philbogle** [comment]
LGTM assuming a manual testing pass.

**Pull request merged**